### PR TITLE
Added third argument to specify publishing frequency

### PR DESCRIPTION
**Pull request by adnanmunawar**
Hello,

I myself needed this change and I have read several posts online for people requesting this change. I have added room for providing the third argument to the executable that specifies the publishing frequency, which was hardcoded to 1 sec before. As can be seen from the code, the third argument can even be not provided and the code will use a default publishing frequency of 500 Hz

### CODE REVIEW

**Comment by tfoote** [comment]
parallel branch request #61

**Comment by tfoote** [comment]
@ros-pull-request-builder

File: tf/src/tf_echo.cpp [hunk 1]
```diff
@@ -66,8 +67,14 @@ int main(int argc, char ** argv)
     printf("Note: This is the transform to get data from target_frame into the source_frame.\n");
     return -1;
   }
+  if (argc = 4)
```
Comment (tfoote) [inline tf/src/tf_echo.cpp]: This needs to be ==

**Comment by adnanmunawar** [comment]
Hello, 
I have merged the 3 commits into one commit, I guess it should be good to go now.
-AM

**Comment by tfoote** [comment]
Thanks, this is close, however it does not catch and report usage for too many arguments. The <3 should also check for >4

**Comment by tfoote** [comment]
replaced in #88

**Pull request closed without merging**